src/main/java/algorithms/leetcode/medium/RemoveStarsFromString.java

```diff
@@ -0,0 +1,43 @@
+/*
+2390. Removing Stars From a String
+MEDIUM
+
+You are given a string s, which contains stars *.
+In one operation, you can:
+i) Choose a star in s.
+ii) Remove the closest non-star character to its left, as well as remove the star itself.
+iii) Return the string after all stars have been removed.
+Note:
+The input will be generated such that the operation is always possible.
+It can be shown that the resulting string will always be unique.
+
+Example 1:
+Input: s = "leet**cod*e"
+Output: "lecoe"
+Explanation: Performing the removals from left to right:
+- The closest character to the 1st star is 't' in "leet**cod*e". s becomes "lee*cod*e".
+- The closest character to the 2nd star is 'e' in "lee*cod*e". s becomes "lecod*e".
+- The closest character to the 3rd star is 'd' in "lecod*e". s becomes "lecoe".
+There are no more stars, so we return "lecoe".
+
+Example 2:
+Input: s = "erase*****"
+Output: ""
+Explanation: The entire string is removed, so we return an empty string.
+ */
+
+package algorithms.leetcode.medium;
+
+public class RemoveStarsFromString {
+    public static String removeStars(String s) {
```

Comment:
Stack is not getting used, so why this problem is under stack problems?

Reply:
Though this problem doesn't use Stack, the _**deleteCharAt()**_ function is same as pop() function and **_append()_** function is same as push()